Resources/Prototypes/Loadouts/items.yml

```diff
@@ -242,3 +288,17 @@
   cost: 3
   items:
     - SolCommonTranslator
+
+- type: loadout
+  id: LoadoutItemLunchboxGenericFilledRandom
+  category: Items
+  cost: 3
+  items:
+    - LunchboxGenericFilledRandom
```

Comment:
This is actually going to cause problems, but not in a way that I'm going to ask you to remove it. On the contrary, I'm going to ask something different of you entirely. Lunchboxes are now the **largest** item to-date being added to Loadouts. They occupy a 2x4 inventory slot. There was already a problem before where certain jobs have so much shit they spawn with, that everything they buy in their loadout will just get dropped on the floor. And the biggest culprit behind this is actually the Survival Kit. 

So here's my challenge for you. We are going to keep the Lunchbox, in fact I'd GLADLY introduce the idea of having larger items like them be introduced to Loadouts. But in order to keep this, I want you to include in this PR, a full removal of Survival Boxes from all default job spawns. Instead Survival Boxes need to be a Loadout Item, along with things like first aid kits(Let medical staff optionally buy more exotic first aid kits for instance). Probably also make the Epipen and Spacepen into 1 point items you can buy in your loadout. Along with a Nitrogen Survival Box for the Slimepeople & Vox.

Reply:
I'm pretty sure the 2x4 Lunchboxes are a Delta change.


Also, removing survival boxes and making them into loadouts should be in a different PR

Reply:
> I'm pretty sure the 2x4 Lunchboxes are a Delta change.
> 
> Also, removing survival boxes and making them into loadouts should be in a different PR

We kind of have no choice if this PR includes loadout lunchboxes. And I'm not going to merge large items like lunchboxes if the survival box is still a default automatic spawn clutter.

Reply:
Since removing survival boxes from job spawns is such a big spawn and I just wanted le drippy clothes, I'm commenting out the lunchboxes for now so they can be added as soon as we do that.